src/utils/__tests__/coordinates.test.ts

```diff
@@ -0,0 +1,119 @@
+import { describe, expect, it } from '@jest/globals'
+
+import { OPENLAYERS_PROJECTION, WSG84_PROJECTION } from '../../constants'
+import { CoordinatesFormat } from '../../fields/CoordinatesInput/constants'
+import { getCoordinates } from '../coordinates'
+
+describe('coordinates', () => {
+  it('getCoordinates Should get DMS coordinates for a dummy lon/lat', async () => {
+    // When
```

Comment:
I don't think there // When // Then are really necessary, unit tests are supposed to be like that.

Reply:
It's a common pattern to structure tests, it helps understand what is being tested, coming from BDD testing.
In this util function test it's not really relevant but in more complex test it is. I admit it's rarely the case in frontend dev.
Cf. https://jossiwolf.medium.com/leveraging-kotlin-for-tests-311bfd8335ee